LondonBritishLibrary/orient/BLorient80.xml

```diff
@@ -54,47 +124,69 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">113</measure>
-                                    <dimensions type="outer" unit="mm">
-                                        <height>117</height>
-                                        <width>76</width>
+                                    <dimensions type="outer" unit="in">
+                                        <height>4.63</height>
+                                        <width>3</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
                             </supportDesc>
                             
                             <layoutDesc>
-                                <layout columns="3" writtenLines="15"/>
+                                <layout writtenLines="15"/>
                             </layoutDesc>
                         </objectDesc>
                         
                         <handDesc>
                             
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is neatly written of <date>19th century</date>.
-                                </desc>
-                                
-                                <date notBefore="1800" notAfter="1900"/>
-                                
+                                <date notBefore="1801" notAfter="1803"/>
+                                <desc>Neatly written.</desc>
                             </handNote>
-                            
                         </handDesc>   
+                        
                         <additions>
-                            
+                            <list>
+                                <item xml:id="e1">
+                                    <locus target="#1 #112 #113"/>
+                                    <desc>Verses of hymns are written in a very crude hand.</desc>
```

Comment:
Thank you! I wonder if it would rather belong to the hand description?

Reply:
My interpretation is that these are not part of the main text.

Reply:
Maybe than one could also encode those hymns as additions?

Reply:
That's what I had first, but then decided to switch to extras :D I don't think that they are complete hymns, just verses (maybe also contained in the main text?), possibly in the margin? Do you have a type for desc to be suggested, if encoded as additions?